docs/v0.1.0/frontend.md

```diff
@@ -0,0 +1,206 @@
+# v0.1.0 Frontend Document
```

Comment:
I don't think we need versioning at this moment.  So just put everything under `docs/` should be fine.

Reply:
Done